partner_priority/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Priority',
+    'summary': "Adds priority to partners.",
+    'author': "Patrick Wilson, Odoo Community Association (OCA)",
+    'website': "https://github.com/OCA/partner-contact",
+    'category': 'Customer Relationship Management',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'base',
```

Comment:
You do not need this depends, base is in [contacts](https://github.com/odoo/odoo/blob/12.0/addons/contacts/__manifest__.py)

Reply:
Done, thanks